spec/mongoid/fields/field_types_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Mongoid::Fields::FieldTypes do
+
+  after do
+    described_class.instance_variable_set(:@mapping, described_class::DEFAULT_MAPPING.dup)
```

Comment:
You are assuming initial state to be the unmodified state.

At a minimum, this logic can be duplicated into a `before` hook, or it can be rewritten to use an `around` hook, read the current mapping, set the default mapping, run the test, restore previous mapping.

Reply:
Done. I have used an around hook which resets the value both before and after.

Reply:
Please mark resolved.